resources/js/Components/Form/CampaignCreateForm.tsx

```diff
@@ -87,20 +89,31 @@ const CampaignCreateForm = ({
                     {errors?.character_id && <ErrorText message={errors?.character_id} />}
                 </StyledGrid>
                 <StyledGrid item xs={12} md={type === 'Edit' ? 12 : 5}>
-                    <Autocomplete
-                        label='Adventure'
-                        id='adventures'
-                        fieldKey='adventures'
-                        onChange={(_, value) => setData('adventure', value)}
-                        defaultValue={data.adventure}
-                        getOptionLabel={(option) => `${option.code} - ${option.title}`}
-                        options={adventures}
-                        resetUrl={
-                            type === 'Create'
-                                ? route('campaign.create')
-                                : route('campaign.index')
-                        }
-                    />
+                    {disabledAdventure ? (
+                        <TextField
```

Comment:
Ternary is not needed, you can just add `disabled={disabledAdventure}` instead of writing extra code

Reply:
i also don't like the prop name, why not something like `readOnly`

Reply:
also you don't really need this prop, isn't there editData or onCloseDrawer to determine if it's in edit mode